### ISSUE TITLE
Add support for `render()` outside of `each()` data-query

### ISSUE THREAD

**Issue by astoilkov** [astoilkov/jsblocks#83]
As per suggestion: https://github.com/astoilkov/jsblocks/issues/58

**Comment by henrikekblad** [comment]
+1

**Comment by Kanaye** [comment]
I read the suggestion and I see a problem as far as I understeand the issue.
If you have an expression in an images source like so:
```html
<img src="{{myImage}}" />
```
the problem (at least for me) is that the browser would have the error that ``{{myImage}}`` is not a correct url. An other case might be a mobile device where you don't want some images to load to smaller your apps/pages traffic.
something like that:
```html
<img src="big-image.jpg" data-query="render(!isMobile)" />
```
The problem is, that the browser would have parsed the attributes and started the request *before* the dom is ready.
So we have no chance to remove that image before the browser starts the request.

Tell me if there are other cases an element should be removed instead of hidden.

I saw the ``attr()``-data-query is currently not capable to set the ``src`` attribute.
Maybe we should implement that, cause the ``attr`` data-query combined with the ``visible`` data-query would be my suggstion for the described case.

**Comment by astoilkov** [comment]
Yes. I thought the attr data-query works correctly. Otherwise it really doesn't make sense.

**Comment by Kanaye** [comment]
short update:
The ``attr`` data-query works as expected with the ``src`` attribute.
For some reason it fails on one of my computers in chrome, but seems to be an issue with the installation and not with jsblocks.
